web_monitoring/diffing_server.py

```diff
@@ -115,8 +115,35 @@ def options(self):
 class DiffHandler(BaseHandler):
     # subclass must define `differs` attribute
 
+    # Compute our own ETag header values.
+    # We're not actually hashing content for this, since that is expensive.
+    def compute_etag(self):
+        query_params = {k: v[-1].decode() for k, v in
+            self.request.arguments.items()}
+        etag = str('W/"' + hashlib.sha256(
```

Comment:
Can you note the meaning of `W/“` (weak, which seems like a good choice 👍) and why we chose it here for people who are not deeply familiar with ETag headers?